pyvista/core/filters/data_set.py

```diff
@@ -5132,3 +5132,44 @@ def reflect(
             inplace=inplace,
             progress_bar=progress_bar,
         )
+
+    def integrate_data(self, progress_bar=False):
+        """Integrate point and cell data.
```

Comment:
It would be good to explicit the algorithm of the integral method. Can we specify the type of algorithm?